src/components/Pages/EventList/EventListEntry.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import moment from 'moment-timezone';
+import { useIntl } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import { useMobileQuery } from 'js/helpers/reactMediaQueries.js';
+import { formatTime, formatHours } from 'js/helpers/cleanData';
+import { events as i18n } from 'js/i18n/definitions';
+
+const EventDateCalendar = ({date}) => {
+  const intl = useIntl();
+  const isMobile = useMobileQuery()
+  
+  moment.locale(intl.locale);
```

Comment:
just a note that I updated https://github.com/cityofaustin/techstack/issues/3897 to include the fact that we're using moment.js in components as well (if we're going to try to come up with a consolidated time solution at some point)